examples/chef/efr32/src/AppTask.cpp

```diff
@@ -76,29 +77,29 @@ void OnTriggerIdentifyEffect(Identify * identify)
 {
     sIdentifyEffect = identify->mCurrentEffectIdentifier;
 
-    if (identify->mCurrentEffectIdentifier == EMBER_ZCL_IDENTIFY_EFFECT_IDENTIFIER_CHANNEL_CHANGE)
+    if (identify->mCurrentEffectIdentifier == Clusters::Identify::EffectIdentifierEnum::kChannelChange)
     {
         ChipLogProgress(Zcl, "IDENTIFY_EFFECT_IDENTIFIER_CHANNEL_CHANGE - Not supported, use effect varriant %d",
-                        identify->mEffectVariant);
-        sIdentifyEffect = static_cast<EmberAfIdentifyEffectIdentifier>(identify->mEffectVariant);
+                        (uint8_t) identify->mEffectVariant);
+        sIdentifyEffect = static_cast<Clusters::Identify::EffectIdentifierEnum>(identify->mEffectVariant);
```

Comment:
This is odd.  Effect identifier and effect variant are _NOT_ the same thing.  This code really doesn't make sense.

Please file a followup issue on this and ping @andy31415 to figure out who owns this code.

Reply:
Yes this is a bit strange but it works here for the purpose it serves. I believe this was written by @pankore .

Reply:
https://github.com/project-chip/connectedhomeip/issues/26797